ProxyAppsSuite/tests/ember.yaml

```diff
@@ -0,0 +1,470 @@
+_base:
+    summary: Large scale modeling of high-performance interconnects
+    scheduler: slurm
+    slurm:
+        num_nodes: 2
+
+_build:
+    inherits_from: _base
+    build:
+        modules: [gcc]
+        source_path: ember
+        cmds:
+            # UCX
+            - 'module load friendly-testing openmpi/3.1.5'
+            - 'tar xvfz ucx-1.8.0.tar.gz &> ucx-build.log'
```

Comment:
make this line (line 15) `- 'tar xfz ucx-1.8.0.tar.gz` - don't redirect. I tried setting up redirection everywhere, removed it, but forgot this line.